src/cascadia/TerminalApp/CommandPalette.xaml

```diff
@@ -482,6 +388,8 @@
 
             <ListView x:Name="_filteredActionsView"
                       Grid.Row="2"
+                      MinHeight="8"

```

Comment:
I suppose here's a thought - why do we need this `MinHeight` at all? Setting it to 0 seems to mitigate the crash, and I don't know what it's needed for. At least, my eyes aren't sensitive enough to see what this changes...